.gitignore

```diff
@@ -1,4 +1,6 @@
+/.cache
 /.vs
+/.vscode
```

Comment:
Please put these in your global gitignore, and not here. Your choice of IDE is a private business :D